src/js/big-play-button.js

```diff
@@ -1,22 +1,41 @@
+/**
+ * @file big-play-button.js
+ */
 import Button from './button.js';
 import Component from './component.js';
 
-/* Big Play Button
-================================================================================ */
+/ * Big Play Button
```

Comment:
@mboles FYI, the find replace wasn't 100% safe. It added a comment-breaking space in a few places like this. I cleaned them up, but just a heads up in case you use that again.